packages/math-input/src/components/keypad/button.tsx

```diff
@@ -58,7 +58,7 @@ const styles = StyleSheet.create({
     },
 });
 
-type Props = {
+export type Props = {
```

Comment:
I was talking to the FE Infra team the other day and they were saying it's becoming an anti-pattern to export props like this. Instead, we should use the following construct to get the components Props type. 

```js
JSX.LibraryManagedAttributes<
    typeof MyComponent,
    React.ComponentProps<typeof MyComponent>
>
```

Reply:
`React.ComponentProps<typeof MyComponent>`?